hooks/mari/publish_mipmaps.py

```diff
@@ -232,3 +362,96 @@ def _valid_for_mipmap_multiimage(self, target_path):
                                 "'{0}'".format(os.path.splitext(target_path)[-1]))
             return False
         return any((_target.supports("mipmap"), _target.supports("multiimage")))
+
+
+    def undo(self, task_settings, item):
+        """
+        Execute the undo method. This method will
+        delete the files that have been copied to the disk
+        it will also delete any PublishedFile entity that got created due to the publish.
+
+        :param task_settings: Dictionary of Settings. The keys are strings, matching
+            the keys returned in the task_settings property. The values are `Setting`
+            instances.
+        :param item: Item to process
+        """
+
+        self.logger.info("Cleaning up copied files for %s..." % item.name)
+
+        sg_publish_data_list = item.properties.get("sg_publish_data_list")
+        mipmap_paths = item.properties.get("mipmap_paths")
```

Comment:
This is the only bit edited from super (delete mipmap_paths instead of publish_path) 
Maybe refactor to avoid copying so much? Avoid changing method signature? Not sure.

Reply:
Please see above comment. We should implement and use `item.properties.publish_paths[self.name]` instead.